Formula/pipenv.rb

```diff
@@ -0,0 +1,108 @@
+class Pipenv < Formula
+  include Language::Python::Virtualenv
+
+  desc "Experimental virtualenv, and package manager for Python"
+  homepage "https://github.com/kennethreitz/pipenv"
+  url "https://github.com/kennethreitz/pipenv/archive/v3.2.10.tar.gz"
```

Comment:
Please, upgrade to 3.2.11. `sha256 "4dc7dcea983f0ecd8f3ca16e9249280271558bdffd0cc1bdca3bdf322f365d4e"`